app/lambda/create_health_care_professional/lambda_function.py

```diff
@@ -0,0 +1,58 @@
+import json
+import uuid
+import sys
+
+sys.path.append("../")
+# import boto3
+from lambda_base import LambdaBase
+from models import HealthCareProfessional
+
+
+class CreateHealthCareProfessionalLambda(LambdaBase):
+    def __init__(self): # implementation-specific args and/or kwargs
+        pass
+
+    def putItem(self, HealthCareProfessionalName):
+        HealthCareProfessionalId = "h-"+uuid.uuid4().hex 
+        info = {'HealthCareProfessionalId': HealthCareProfessionalId, 'HealthCareProfessionalName':  HealthCareProfessionalName}
+        HealthCareProfessional().createHealthCareProfessional(info)
+        return HealthCareProfessionalId
+        # client = boto3.resource('dynamodb')
```

Comment:
Please remove the statements that are commented out if not needed

Reply:
Done.